src/components/Table/index.tsx

```diff
@@ -18,4 +23,12 @@ const Table = (props: TableProps) => {
   )
 }
 
+export const mdxTableComponents: MDXProviderComponentsProp = {
+  table: Table,
+  th: ({ align, ...rest }) => <Th textAlign={align} {...rest} />,
+  td: ({ align, ...rest }) => <Td textAlign={align} {...rest} />,
```

Comment:
@pettinarip From what I saw the syntax is automatically converted to the `align` prop and passed to the `th` and all the `td` elements in the respective column, thereby exposing `align` to the render functions

Reply:
Awesome!! that is great!

Reply:
@pettinarip would you like to use this in the templates so it can be viewed in action? Or wait for another PR?

I should also see about making or rewriting a story that uses the MDX provider 🤔